libs/product/state/src/reducers/configurable-product-entities/configurable-product-entity.ts

```diff
@@ -1,11 +1,29 @@
 import { DaffProduct } from '@daffodil/product';
 
+/**
+ * Describes how configurable product configured attributes are stored in rxjs entity state.
```

Comment:
are stored in `the` rxjs entity state?

Reply:
I prefer "in rxjs entity state" here. Just thinking of it if was "stored in state" vs "stored in the state". "Stored in state" seems to be the more common usage